content/en/metrics/custom_metrics/guide/tag-governance-drop-metrics-using-specific-tags.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Tag Governance - Drop Metrics Using Specific Tags
+kind: guide
+further_reading:
+- link: "https://vector.dev/docs/setup/going-to-prod/"
+  tag: "Documentation"
+  text: "Deploying Vector to production"
+- link: "https://vector.dev/docs/reference/configuration/sources/datadog_agent/ "
+  tag: "Documentation"
+  text: "Datadog agent as a source for Vector"
+---
+
+## Overview
+
+Custom metrics provide visibility into all facets of your business -- from application performance, infrastructure health and business KPIs. In order to govern your custom metric volumes, Datadog offers several tools for cost visibility and control, such as [estimated real-time custom metric usage][1], [usage attribution][2], and [Metrics without Limits][3]™.
+
+These tools use metrics tags to provide visibility into metrics that have been ingested into Datadog. This helps you monitor the number of custom metrics associated with any tag. For example, you can use tags to see how many custom metrics a particular application team is generating. However, custom metrics can be submitted without proper tagging; making it harder to understand your overall volume of custom metrics. It’s also harder, without proper tagging, to attribute the metrics back to the specific team, service, or application that is generating them.
+
+## Prerequisites
+
+To filter out custom metrics based on tags, use [Vector][4] to drop them before they get ingested into Datadog. Vector, an open source tool optimized for observability pipelines, or its enterprise solution, Datadog Observability Pipelines, lets you control which custom metrics are sent to Datadog.
+
+This guide assumes that you already have a Vector pipeline set-up. If you’re not familiar with Vector, see [Setting up Vector][5] and [Vector Remap Language][6] for more information.
```

Comment:
Removed link to learning course because its been temporarily taken down for maintenance.